en/manual/scripts/serialization.md

```diff
@@ -0,0 +1,145 @@
+﻿# Serialization
+
+<span class="label label-doc-level">Beginner</span>
+<span class="label label-doc-audience">Programmer</span>
+
+The editor and serialization system uses four attributes to determine what is serialized and visible in the editor.
```

Comment:
There are actually five attributes"DisplayAttribute" which is used to stop the property from showing in the property grid or altering it there but still have it included in yaml and binary serialization

Reply:
Yep, moved display over into public properties and fields as this is not related to serialization, but more editor styling.

Reply:
browsable affects the editor serialization.
if its not that important lets leave it out

Reply:
This is the only read reference I found:
https://github.com/stride3d/stride/blob/master/sources/presentation/Stride.Core.Presentation.Quantum/ViewModels/NodeViewModelFactory.cs#L86-L96
It just hides it from the editor, doesn't affect serialization, unless I am mistaken ?

Reply:
allright